tracpro/polls/tests/test_charts.py

```diff
@@ -247,6 +247,8 @@ def test_single_pollrun_open(self):
         self.assertEqual(summary_table, None)
 
     def test_single_pollrun_numeric(self):
+        # Make answers numeric
+        models.Answer.objects.update(category='numeric')
```

Comment:
We'll want to update this test per an earlier comment to update the Questions to numeric.